homeassistant/components/aqara/strings.select.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
This PR currently has no select platform, thus this translation file can be removed from this PR.